test/verify/check-apps

```diff
@@ -177,7 +182,7 @@ class TestApps(PackageCase):
         b.wait_present(".app-description:contains('DESCRIPTION:de')")
         b.wait_not_present(".app-description:contains('DESCRIPTION:none')")
 
-        set_lang("ja-ja")
```

Comment:
Hmm how is this change related?

Reply:
`ja-ja` is not a valid locale. It somehow worked before, but is broken now. `ja-jp` is the expected locale. Originally fixed here: https://github.com/cockpit-project/cockpit/commit/a395a4a5155e62b24cdb8afa3697618c31282e1f

I could split it to separate commit if you would prefer that.

Reply:
It's fine - I just thought it was a typo.